### PR TITLE
tests: Fix tests on RADV

### PR DESCRIPTION
**Pull request by spencer-lunarg**
some edge cases that were failing on RADV

### CODE REVIEW

File: tests/unit/shader_object.cpp [hunk 1]
```diff
@@ -6703,6 +6703,8 @@ TEST_F(NegativeShaderObject, MissingComputeFullSubgroups) {
     TEST_DESCRIPTION("Create shader with invalid flags when computeFullSubgroups is not enabled.");
 
     m_errorMonitor->SetDesiredError("VUID-VkShaderCreateInfoEXT-flags-09405");
+    // if subgroup size is not 32
+    m_errorMonitor->SetUnexpectedError("VUID-VkShaderCreateInfoEXT-flags-08417");
```
Comment (artem-lunarg) [inline tests/unit/shader_object.cpp:6707]: If this test is about subgroups how important to specify valid number here? The popular options on the  market for the width of execution unit: 32 and 64 (nvidia/amd, and I guess intel too). Vulkan database shows there are mobile devices that advertise 128.

Comment (spencer-lunarg) [reply tests/unit/shader_object.cpp:6707]: I'm just going to make it 64, and query if it is over 64 we skip

Comment (artem-lunarg) [reply tests/unit/shader_object.cpp:6707]: good idea

Comment (spencer-lunarg) [reply tests/unit/shader_object.cpp]: so found we had a more dynamic way to do this in another test

**Review by artem-lunarg — APPROVED** [review]
